docker/Dockerfile

```diff
@@ -34,6 +35,11 @@ RUN apt-get update && \
         tmux && \
     rm -rf /var/lib/apt/lists/*
 
+RUN pip install --upgrade setuptools pip
+RUN pip install \
+        recommonmark==0.4 \
+        sphinx==1.5.6
```

Comment:
@apojomovsky any powerful reason to get hooked to these specific versions?

Reply:
Yes, there's some incompatibility across `recommonmark` and some newer versions of `sphinx`, so what I did here is to freeze `recommonmark` to its latest version to the date, and freeze `sphinx` to the latest known to work version.